demo/psa/prov-verif-e2e/README.md

```diff
@@ -17,19 +17,75 @@ end to end PSA Demonstration.
 
 ## Creation of PSA Endorsements
 
-* The Supply Chain Endorsements used in provisioning pipeline setup below are created using the instructions given [here](./COCLI_README.md) Use these instructions if one wants to create their own Endorsements or modify existing ones.
+First create new Concise Module Identifiers (CoMID's) and use them in creating Concise Reference Integrity Manifests (CoRIM's) using reference templates located under `docs/demo/psa/prov-verif-e2e/data/templates` to provision them in Veraison Verification Service.
 
-## Provisioning pipeline setup
+More details about CoRIM and CoMID can be found [here](datatracker.ietf.org/doc/draft-ietf-rats-corim/)
 
-* Install `cocli` tool using following command
+### Initial Setup
 
-In a new shell session
+Do this ONLY if it is not done already:
+
+In a new bourne shell session
+
+```shell
+export TOPDIR=$(pwd)
```

Comment:
It's not clear what  `$(pwd)` is here. You should specify where you  expect the user to be (e.g. the root dir of the repo, or create a new working directory that several repos will be cloned to).

Reply:
@setrofim is it not clear that we are expecting the user to be in its default folder location when the shell is  created, as $pwd will give it the same default location and TOPDIR will be set accordingly with this command?